config/globals.js

```diff
@@ -47,6 +47,7 @@ const globals = {
 	SC_DOMAIN: process.env.SC_DOMAIN || 'localhost',
 	SC_THEME: process.env.SC_THEME || 'default',
 	SC_TITLE: process.env.SC_TITLE || 'HPI Schul-Cloud',
+	SC_TIME_ZONE: process.env.SC_TIME_ZONE || 'CEST',
```

Comment:
Not needed here. The default value of SC_TIME_ZONE should be defined in the default.js.
Later on we will get the value by using Configuration.get("SC_TIME_ZONE")

Reply:
ok ;), deleted